src/main/resources/application.yaml

```diff
@@ -92,28 +94,29 @@ hapi:
     #    default_pretty_print: true
     #    default_page_size: 20
     #    delete_expunge_enabled: true
-    #    enable_repository_validating_interceptor: false
+    #    enable_repository_validating_interceptor: true
```

Comment:
This line should match the upstream repo.

Reply:
Yes I will update it.